src/components/EntityOpsLevelMaturityContent.tsx

```diff
@@ -156,6 +156,7 @@ export const EntityOpsLevelMaturityContent = () => {
           </Grid>
           <Grid item>
             <CheckResultsByLevel
+              opslevelUrl={service?.htmlUrl}
```

Comment:
this currently uses the service's base URL and then attaches /rubric or whatever, resulting in something like "http://app.opslevel.local:3000/services/service-f1/rubrics" or "http://app.opslevel.local:3000/services/service-f1/teams/bu-frontend"; both not valid URLs. can we reuse the base URL from api.ts?

Reply:
Yep, I can plug that in now